docs/examples/infer-orientation-daily-peak.py

```diff
@@ -0,0 +1,90 @@
+"""
+Infer System Orientation (Azimuth & Tilt) using Daily Peak
+==========================================================
```

Comment:
The titles for these two orientation examples end up rendered awkwardly because of their length.  What do you think about something like `Infer Array Tilt/Azimuth - Daily Peak` and `Infer Array Tilt/Azimuth - PVWatts`?

Reply:
Changed to recommended titles in https://github.com/pvlib/pvanalytics/pull/146/commits/7ee573577bc704d95d49e2b0b8a02d5688184c47